src/__stories__/00-Uncontrolled/08-NestedObject.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Table } from 'react-bootstrap';
+import { DatatableWrapper } from '../../components/DatatableWrapper';
+import { TableColumnType } from '../../helpers/types';
+import { StoryColumnType } from '../resources/types';
+import NESTED_TABLE_DATA from '../resources/nested-story-data.json';
+import { TableHeader } from '../../components/TableHeader';
+import { TableBody } from '../../components/TableBody';
+
+// @@@SNIPSTART CustomRowStyle
```

Comment:
Could we update this `CustomRowStyle` into `NestedObject`? This part is used to synchronize the snippets in https://imballinst.github.io/react-bs-datatable/examples/, later on I can create a new page consisting this one.